src/Symfony/Component/Filesystem/composer.json

```
@@ -19,6 +19,9 @@
         "php": ">=7.1.3",
         "symfony/polyfill-ctype": "~1.8"
     },
+    "require-dev": {
+        "symfony/finder": "^4.0|^5.0"
```

```suggestion
        "symfony/finder": "^4.4|^5.0"
```
4.0 is dead an buried. 😃